### PR TITLE
Removed Unused import

### PR DESCRIPTION
**Pull request by Yash-Varshney**

### CODE REVIEW

**Comment by Yash-Varshney** [comment]
@pdxjohnny  what just happened here ? I expected it to work smooth as I have removed few unused imports .

**Comment by pdxjohnny** [comment]
@Yash-Varshney It looks like some of the imports you removed we're false positives, they are actually used. Next steps here would be to read the GitHub actions logs and see which ones need to be added back in. Then, for the ones that you had to add back in but LGTM thinks should be removed. You'll want to figure out how what the correct comment is to add to the line causing the false positive. LGTM will have a way to ignore issues for a line.

File: lgtm.yaml [hunk 1]
```diff
@@ -3,6 +3,8 @@ path_classifiers:
     - scripts/docs.py
   test:
     - tests
+  queries:
+    - exclude: py/import-and-import-from
```
Comment (pdxjohnny) [inline lgtm.yaml:7]: This makes it so all unused imports are ignored. You want to remove this, and add comments on the individual lines themselves which were false positives.

image

Comment (Yash-Varshney) [reply lgtm.yaml:7]: @pdxjohnny this actually does not ignore all the unused imports, in LGTM there is an error which is raised when we use something like - 
` import pathlib
from pathlib import Path`
this error will be raised even when we use `Path`
LGTM asks us to do the above thing like this - 
`Remove the from xxx import yyy statement. Add yyy = xxx.yyy if required.`

Should I still make a comment on particular file or is it okay to exclude these type of errors.

Comment (pdxjohnny) [reply lgtm.yaml:7]: Ah yes. We still don't want to ignore those, we should be fixing them instead

**Comment by pdxjohnny** [comment]
Thanks Yash!!!

**Pull request merged**